### ISSUE TITLE
Confused specification of _ID_BenefitSurtax_crt and associated logic

### ISSUE THREAD

**Issue by martinholmer** [open-source-economics/Tax-Calculator#653]
The reform parameter (which is not part of current tax law) ```_ID_BenefitSurtax_crt``` is defined in a confusing way.  Here is its specification in the ```current_law_policy.json``` file:
```
"_ID_BenefitSurtax_crt": {
    "long_name": "Credit on itemized deduction benefit surtax (percent of AGI)",
     "description": "The surtax on specified itemized deductions applies to benefits in excess of this percent of AGI. In http://www.nber.org/papers/w16921, Feldstein, Feenberg, and MacGuineas propose a credit of 2% of AGI against a 100% tax rate; in their proposal, however, a broader set of tax benefits, including the employer provided health exclusion, would be taxed.",
    "irs_ref": "",
    "notes": "",
    "start_year": 2013,
    "col_var": "",
    "row_var": "FLPDYR",
    "row_label": ["2013"],
    "cpi_inflated": false,
    "col_label": "",
    "value": [1.0]
},
```

And in the ```functions.py``` file, the logic of applying such a surtax is skipped if the benefit surtax credit is not equal to one percent, as can be seen here:
```
def BenefitSurtax(calc):
    """
    BenefitSurtax function: ...
    """
    if calc.policy.ID_BenefitSurtax_crt != 1.:
        # do the surtax calculations
```

This confusion could be eliminated if all the ```percent of AGI``` wording was removed, or replaced by ```decimal fraction of AGI``` wording.

But even after changing the wording, shouldn't the ```functions.py`` code be:
```
if calc.policy.ID_BenefitSurtax_crt < 1.:```
```
Certainly neither the TaxBrain webapp nor the taxcalc package balk at setting ```_ID_BenefitSurtax_crt``` equal to a number larger than one.

**Comment by martinholmer** [comment]
What do people think about issue #653?

@MattHJensen @feenberg @Amy-Xu @talumbau @GoFroggyRun

**Comment by MattHJensen** [comment]
@martinholmer, `decimal fraction` does seem like an improvement, and probably one that we should adopt elsewhere in parameter descriptions. I believe `if calc.policy.ID_BenefitSurtax_crt < 1.` is an improvement as well, but @amy-xu, could you try to recall if there was a reason for using the `if calc.policy.ID_BenefitSurtax_crt != 1.` structure instead?

**Comment by Amy-Xu** [comment]
What I remember from previous conversations is, we should leave the options to our users if we do have the data for those calculations. In this case, even though greater than 1 values wouldn't make much sense, it's still possible to get a result from taxcalc. I have tried with a cap rate of 2 and tax rate of 10, and got some "reasonable" revenue changes.

**Comment by martinholmer** [comment]
Amy said:
> What I remember from previous conversations is that we should leave the options to our users if we do have the data for those calculations. In this case, even though greater than 1 values wouldn't make much sense, it's still possible to get a result from taxcalc. I have tried with a cap rate of 2 and tax rate of 10, and got some "reasonable" revenue changes.

OK, we can leave the ```if calc.policy.ID_BenefitSurtax_crt != 1``` code, but the characterization (in the ```current_law_policy.json``` file) of the parameters as being a **percent of AGI** seems like it needs to be changed.

@Amy-Xu @MattHJensen @feenberg

**Comment by Amy-Xu** [comment]
Definitely. Do you prefer me opening a PR for this?

**Comment by martinholmer** [comment]
@Amy-Xu said:
> Definitely. Do you prefer me opening a PR for this?

I'll be happy to do this as I'm waiting on some other things.
All I need to do is change the wording in the ```current_law_policy.json``` file, right?

**Comment by Amy-Xu** [comment]
Yep I think so.

**Comment by martinholmer** [comment]
@Amy-Xu and @MattHJensen,  Does pull request #664 look okay as a way to resolve issue #653?

**Comment by martinholmer** [comment]
Issue #653 has been resolved by pull request #664.
